op-program/host/prefetcher/retry.go

```diff
@@ -12,6 +12,7 @@ import (
 )
 
 const maxAttempts = math.MaxInt // Succeed or die trying
+const maxAttemptsExpensiveOp = 2
```

Comment:
I'm not sure this is safe.  It's definitely wrong for `OutputByRoot` since we don't have a fallback for that - we need to keep retrying until we get the data.

With the way that `prefetcher.prefetch` is currently written we'd try to use ExecutionWitness of GetProof and if they fail would return an error without trying to fetch via debug_dbGet.  That error would bubble up and cause the op-program host to fail with an error.

I'd say either we don't retry these functions at all and fix it so that if they return an error we switch to the fallback, or we retry forever like the other functions.

Reply:
Makes sense - the original justification for OutputByRoot was that it should match AccountProof since they both call get proof, but I think just keeping that behavior untouched and removing retry from the experimental methods is better.

I removed retry behavior, replace panic with error, added graceful retry to experimental methods: 
ca94656